services/web-server/src/servers/createApp.js

```diff
@@ -43,8 +70,26 @@ module.exports = async ({ cfg, strategies }) => {
     );
   }
 
-  passport.serializeUser((user, done) => done(null, user));
-  passport.deserializeUser((obj, done) => done(null, obj));
+  passport.serializeUser((user, done) => {
+    const isGithub = user.profile.provider === 'github';
+    const userId = isGithub ? user.profile.id : user.profile.user_id;
+
+    return done(null, {
+      identityProviderId: isGithub ? 'github' : 'mozilla-auth0',
+      userId,
+    });
```

Comment:
Rather than special-case github here (since we'll eventually have other strategies..), could this just include the identity string (from `user.identity`)?

Reply:
Good idea. Not sure why I didn't think about that :)